test/query_test.py

```diff
@@ -90,6 +90,21 @@ def test_derived_p0(query_derived, query_atnf):
 
     assert errval == derval
 
+@pytest.mark.enable_socket
+def test_derived_ecliptic(query_derived, query_atnf):
+    """
+    Test the derived ecliptic longitude and latitude.
+    """
+
+    elong = query_derived.get_pulsar('TEST1')['ELONG'][0]
+    elongatnf = query_atnf.get_pulsar('TEST1')['ELONG'][0]
+
+    assert abs(elong-elongatnf) < 1e-5
```

Comment:
This is a good test, but why 1e-5? What is the usual precision of the `elong` and `elat` values in the catalogue?

Reply:
The `1e-5` was just put in because it worked (`1e-6` fails). I'm not actually sure what consistency to expect between how `psrcat` calculates the ecliptic coordinates and how `psrqpy` does it using astropy. It might be that we're being more accurate, or they're both correct, but using slightly different conventions somewhere.